app/views/assistants/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Assistants in <%= @course.title %></h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+    </tr>
+  </thead>
+
+  <tbody>
+  <% @assistants.each do |assistant| %>
+    <tr>
+      <td><%= assistant.username %></td>
```

Comment:
Little bit permission tuning needed for this to work well, will be done in bigger context in later PR, marked as own story/issue (https://github.com/TMC-teacher-tools/tmc-server/issues/75)